storops/unity/resource/lun.py

```diff
@@ -37,7 +37,21 @@
 log = logging.getLogger(__name__)
 
 
-def prepare_lun_parameters(**kwargs):
+def prepare_lun_parameters(cli=None, **kwargs):
+    @version('<4.3')
+    def make_data_reduction_body(cli=None,
+                                 is_data_reduction_enabled=None):
+        if is_data_reduction_enabled is None:
```

Comment:
You could use `UnityClient.make_body` which will remove the None value key-value pair from the dict. Then there is no need to check `if is_data_reduction_enabled is None`.

Reply:
Done.